DatadogSessionReplay/Sources/Recorder/Utilities/SystemColors.swift

```diff
@@ -0,0 +1,63 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import UIKit
+
+/// Collection of system colors.
+///
+/// Contextual colors are light- and dark-mode sensitive and must be implemented as computed variables,
+/// so they return different values upon `UIUserInterfaceStyle` change.
+///
+/// For older iOS versions that do not support `UIUserInterfaceStyle`, approximate fallbacks are provided.
+/// See https://gist.github.com/ncreated/35bf4d69d83d1a5ab408ff29a77fc9ff for reference when updating this collection.
+internal enum SystemColors {
+    static let clear: CGColor = UIColor.clear.cgColor
+
+    static var tertiarySystemFill: CGColor {
+        if #available(iOS 13.0, *) {
+            return UIColor.tertiarySystemFill.cgColor
+        } else {
+            // Fallback to iOS 16.2 light mode color:
+            return UIColor(red: 118 / 255, green: 118 / 255, blue: 128 / 255, alpha: 1).cgColor
+        }
+    }
+
+    static var tertiarySystemBackground: CGColor {
+        if #available(iOS 13.0, *) {
+            return UIColor.tertiarySystemBackground.cgColor
+        } else {
+            // Fallback to iOS 16.2 light mode color:
+            return UIColor(red: 255 / 255, green: 255 / 255, blue: 255 / 255, alpha: 1).cgColor
+        }
+    }
+
+    static var secondarySystemFill: CGColor {
+        if #available(iOS 13.0, *) {
+            return UIColor.secondarySystemFill.cgColor
+        } else {
+            // Fallback to iOS 16.2 light mode color:
+            return UIColor(red: 120 / 255, green: 120 / 255, blue: 128 / 255, alpha: 1).cgColor
+        }
+    }
+
+    static var tintColor: CGColor {
+        if #available(iOS 15.0, *) {
+            return UIColor.tintColor.cgColor
```

Comment:
Why 15? this one is available since iOS7

Reply:
Indeed, the [Apple doc says iOS7](https://developer.apple.com/documentation/uikit/uiview/1622467-tintcolor), but this seems to not be true:

<img width="838" alt="Screenshot 2023-02-21 at 15 20 37" src="https://user-images.githubusercontent.com/2358722/220370703-3ca5a040-6ca6-4c7a-ac58-43fb041c282c.png">

<img width="496" alt="Screenshot 2023-02-21 at 15 20 05" src="https://user-images.githubusercontent.com/2358722/220370736-36bcdd46-d976-4548-aa33-40efb37daf78.png">

🤷